opal/runtime/opal_params_core.c

```diff
@@ -90,7 +90,7 @@ static bool opal_register_util_done = false;
 /* Basic color options: 30=black, 31=red, 32=green,
  * 33=yellow, 34=blue, 35=magenta, 36=cyan, 37=white
  * https://en.wikipedia.org/wiki/ANSI_escape_code#Colors */
-static char *opal_var_dump_color_string = "name=34,value=32,valid_values=36";
+static char *opal_var_dump_color_string = NULL;
```

Comment:
Because mca_base_var_register is allocating its own storage, and you've now placed the default as a local during initialization, is there any reason to keep this global variable?

Reply:
There's no point, it could be declared in the function itself

Reply:
Is the storage allowed to be on the stack? I'm getting errors if it's placed inside the function.

Reply:
@gkatev what errors are you seeing? I didn't see errors when I ran it